### PR TITLE
Added newsystem flag to be passed to latte

### PR DESCRIPTION
**Pull request by cnegre**
## Purpose

_Pass a flag to latte (newsystem) to tell if there is a new structure to be used in a calculation. Added an example of two consecutive simulations. Renamed examples for consistency_

## Author(s)

_S. J. Plimpton and C. F. A. Negre_

## Backward Compatibility

_Full backward compatibility_

## Implementation Notes

_A new example is provided to verify the new implementation_

## Post Submission Checklist

_Please check the fields below as they are completed_
- [*] The feature or features in this pull request is complete
- [ ] Suitable new documentation files and/or updates to the existing docs are included
- [ ] One or more example input decks are included
- [*] The source code follows the LAMMPS formatting guidelines

## Further Information, Files, and Links

_No additional information_

### CODE REVIEW

File: src/LATTE/fix_latte.cpp [hunk 1]
```diff
@@ -37,8 +37,8 @@ using namespace FixConst;
 extern "C" {
   void latte(int *, int *, double *, int *, int *,
              double *, double *, double *, double *,
-             double *, double *, double *, int*,
-             double *, double *, double *, double *, bool *);
+             double *, double *, double *, int *,
+             double *, double *, double *, double *, int * , bool *);
```
Comment (junghans) [inline src/LATTE/fix_latte.cpp]: If you are changing ABI you need to update the `LATTE_ABIVERSION` in `latte_lib.f90` and the abi check below.
Plus, you need to update the version downloaded in `lib/latte/Install.py` and `cmake/CMakeLists.txt`.

Comment (cnegre) [reply src/LATTE/fix_latte.cpp]: - Change LATTE_ABIVERSION to something like 20180622?
- Ok, I will make a new latte release (v1.2.0) and and update Install.py and CMakeLists.txt.

Comment (cnegre) [reply src/LATTE/fix_latte.cpp]: I changed both Install.py and CMakeLists.txt. I also did a release of latte v1.2.0

File: cmake/CMakeLists.txt [hunk 2]
```diff
@@ -296,7 +296,7 @@ if(PKG_LATTE)
     message(STATUS "LATTE not found - we will build our own")
     include(ExternalProject)
     ExternalProject_Add(latte_build
-      URL https://github.com/lanl/LATTE/archive/v1.1.1.tar.gz
+      URL https://github.com/lanl/LATTE/archive/v1.2.0.tar.gz
       URL_MD5 cb86f1d2473ce00aa61ff6a023154b03
```
Comment (junghans) [inline cmake/CMakeLists.txt]: This checksum needs an update, too!

File: src/LATTE/fix_latte.cpp (hunk 1 above)
Comment (junghans) [reply src/LATTE/fix_latte.cpp]: But`LATTE_ABIVERSION` in v1.2.0 is updated: https://github.com/lanl/LATTE/blob/v1.2.0/src/latte_lib.f90#L65

Comment (akohlmey) [reply src/LATTE/fix_latte.cpp]: @junghans I think that should read, that ABIVERSION was **not** updated. It points to february 7th, 2018. The ABIVERSION commit seems to have happened **after** the release was tagged.

Comment (junghans) [reply src/LATTE/fix_latte.cpp]: The abi check in line 58 below needs an update!

**Comment by cnegre** [comment]
Done with changing abi check in line 58!

File: src/LATTE/fix_latte.cpp (hunk 1 above)
Comment (junghans) [reply src/LATTE/fix_latte.cpp]: @akohlmey yes, that was a typo! @cnegre now we just new another LATTE release with the new abi version in it ;-)

File: lib/latte/Install.py [hunk 3]
```diff
@@ -12,12 +12,12 @@
 Syntax from src dir: make lib-latte args="-b"
                  or: make lib-latte args="-p /usr/local/latte"
                  or: make lib-latte args="-m gfortran"
-                 or: make lib-latte args="-b -v 1.1.1"
+                 or: make lib-latte args="-b -v 1.2.0"
```
Comment (junghans) [inline lib/latte/Install.py]: Version update here, too?

**Pull request merged**